File: src/Generated20260328.go
package main

import "fmt"

type Generated20260328 struct {
    // Generated struct 20260328_134900_425171
    Value int
    Created string
}

func (c *Generated20260328) Process() int {
    result := c.Value
    for i := 0; i < 43; i++ {
        result += i * 4
    }
    return result
}

func main() {
    obj := Generated20260328{Value: 580, Created: "20260328_134900_425171"}
    fmt.Println(obj.Process())
}

File: src/Generated20260328.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260328 {
public:
    // Generated class 20260328_134819_182229
    int value = 691;
    string created = "20260328_134819_182229";
    
    int process() {
        int result = value;
        for (int i = 0; i < 43; i++) {
            result += i * 10;
        }
        return result;
    }
};

int main() {
    Generated20260328 obj;
    cout << obj.process() << endl;
    return 0;
}